scalableminds/webknossos
# Add warning if opacity of volume layer is zero

### Steps to test:
- Open an annotation with a volume layer
- Using the slider, set the opacity of a volume layer to zero
- Then a yellow (/orange) warning icon should appear next to the opacity label
- It should disappear when the opacity is set to a value higher than 0

### TODOs:
- [x] layout
- [x] code structure

### Issues:
- fixes #7407 

------
(Please delete unneeded items, merge only when none are left open)
